cyclonedx/parser/environment.py

```diff
@@ -74,8 +72,5 @@ def __init__(self) -> None:
             self._components.append(c)
 
     @staticmethod
-    def _get_metadata_for_package(package_name: str) -> email.message.Message:
-        if sys.version_info >= (3, 8, 0):
-            return metadata(package_name)
```

Comment:
this was the same for py38 and below.
i suspect some internal implementation got lost on the way ...